app/workers/commit_monitor_handlers/commit/bugzilla_common/bugzilla_common.rb

```diff
@@ -0,0 +1,33 @@
+module CommitMonitorHandlers
+  module Commit
+    module BugzillaCommon
+      def product
+        Settings.commit_monitor.bugzilla_product
+      end
+
+      def with_bug(bug_id)
+        return unless block_given?
+        MiqToolsServices::Bugzilla.call do
+          output = ActiveBugzilla::Bug.find(:product => product, :id => bug_id)
+          if output.empty?
+            logger.error "Unable to write for bug id #{bug_id}: Not a '#{product}' bug."
```

Comment:
This error message doesn't make sense here...maybe raise an exception instead and let the caller handle it?  Another option is to just yield output.first, and let the caller handle nil itself.